boot/freeldr/bootsect/btrfs.S

```diff
@@ -0,0 +1,790 @@
+/*
+ * COPYRIGHT:       See COPYING in the top level directory
+ * PROJECT:         ReactOS Bootsector
+ * FILE:            boot/freeldr/bootsect/btrfs.S
+ * PURPOSE:         BTRFS boot sector
+ * PROGRAMMERS:     Victor Perevertkin
+ */
```

Comment:
Since this is a new file, let's take an opportunity to use the new file headers:
https://www.reactos.org/wiki/Coding_Style#File_Structure

Reply:
Such header goes to both headers (*.h) and code files (*.c) or just to code files?

Reply:
Both.